### ISSUE TITLE
Support notebook parameters

### ISSUE THREAD

**Issue by MichaelHatherly** [PumasAI/QuartoNotebookRunner.jl#36]
https://quarto.org/docs/computations/parameters.html

**Comment by MichaelHatherly** [comment]
Behaviour of https://papermill.readthedocs.io/en/latest/usage-parameterize.html#how-parameters-work seems most reasonable to implement.

**Comment by jkrumbiegel** [comment]
I think I would do a mixture of both behaviors. I actually like the yaml header syntax better than the tagged cell, because tagging a cell is a new kind of behavior vs. just adding more configuration at the top. And because papermill itself notes that one cannot use the parameter settings like code:

<img width="838" alt="image" src="https://github.com/PumasAI/QuartoNotebookRunner.jl/assets/22495855/3974e1e1-2741-4b68-9a60-875eb24e757d">

If there's a code cell, people will be tempted to write arbitrary Julia expressions, while the parameters features is just for basic datatypes anyway, as it's specified through YAML.

Unlike knitr, I wouldn't introduce a `params` variable but just use the param names directly as variables, like in python.

So to sum up, something like this:


````md
---
params:
    a: 1
    b: 2
---

```{julia}
a + b
```
# 3

**Comment by MichaelHatherly** [comment]
Agreed, that seems like reasonable behaviour to me.
